benchmarks/bench.sh

```diff
@@ -631,20 +631,22 @@ data_tpch() {
 
 # Points to TPCDS data generation instructions
 data_tpcds() {
-    TPCDS_DIR="${DATA_DIR}"
-
-    # Check if TPCDS data directory exists
-    if [ ! -d "${TPCDS_DIR}" ]; then
-        echo ""
-        echo "For TPC-DS data generation, please clone the datafusion-benchmarks repository:"
-        echo "  git clone https://github.com/apache/datafusion-benchmarks"
-        echo ""
-        return 1
+    TPCDS_DIR="${DATA_DIR}/tpcds_sf1"
+
+    # Check if `web_site.parquet` exists in the TPCDS data directory to verify data presence
+    echo "Checking TPC-DS data directory: ${TPCDS_DIR}"
+    if [ ! -f "${TPCDS_DIR}/web_site.parquet" ]; then
```

Comment:
Extract a variable for `"${TPCDS_DIR}/web_site.parquet"` at line 44 ? And reuse it in `data_tpcds()` and `run_tpcds()`